components/locale_core/src/langid.rs

```diff
@@ -27,7 +27,8 @@ use writeable::Writeable;
 /// `_` separators to `-` and adjusting casing to conform to the Unicode standard.
 ///
 /// Any bogus subtags will cause the parsing to fail with an error.
-/// No subtag validation is performed.
+///
+/// No subtag validation or alias resolution is performed; use `LocaleCanonicalizer`.
```

Comment:
```suggestion
/// For validation and canonicalization, see `LocaleCanonicalizer`.
```

Reply:
I think it is useful to explicitly call out that this function does not perform validation and canonicalization. If you expand the file, there is a list of three levels of conformance, and this function performs one but not the other two.

It is still a bit confusing that the previous line references "bogus subtags" without any definition of what "bogus" means.